packages/common/src/utils/filters.ts

```diff
@@ -323,7 +329,7 @@ export const createDashboardFilterRuleFromField = (
             disabled: !isTemporary,
             label: undefined,
         },
-        includeDefaultValue ? [] : null,
+        value !== undefined ? [value] : [],
```

Comment:
so if we are creating a "is null" operator do we still have to provide it as a value?

operator: NULL
value: [null] << ?

Reply:
Good spot, I mean `value === null`, thank you!